test/services/google_api/add_member_to_google_group_test.rb

```diff
@@ -0,0 +1,39 @@
+require_relative "../../test_helper"
+
+class GoogleAPI::AddMemberToGoogleGroupTest < ActiveSupport::TestCase
+  describe ".call" do
+    it "calls build_directory_service" do
+      expect(GoogleAPI::AddMemberToGoogleGroup).to receive(:build_directory_service)
```

Comment:
btw: THANK YOU SO MUCH for adding a sane mocking library to our app!

Reply:
Weeeee! Yeah idk what I would do without rspecs mocks. They are the only mocking library I have ever used that were intuitive and so clear to use.